batchflow/monitor.py

```diff
@@ -190,6 +198,12 @@ def get_usage(pid=None, **kwargs):
         return process.memory_full_info().uss / (1024 ** 3) # gbytes
 
 
+def get_current_gpus():
+    """ If the `CUDA_VISIBLE_DEVICES` is set, check it and return device numbers. Otherwise, return [0]. """
+    env_variable = os.environ.get('CUDA_VISIBLE_DEVICES', '0')
```

Comment:
Why '0' instead of all availiable GPUs?

Reply:
I think it is more often to use only one GPU even if all of them are available in the current notebook (env variable is not set). In cases of multiple GPU, one should manually pass arguments to monitor initialization